.travis.yml

```diff
@@ -11,25 +10,22 @@ notifications:
 
 # command to install dependencies
 install:
-  - pip install flake8 PyYAML argparse rospkg catkin_pkg defusedxml
-  - sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-key C1CF6E31E6BADE8868B172B4F42ED6FBAB17C654
-  - sudo sh -c 'echo "deb http://packages.ros.org/ros/ubuntu `lsb_release -cs` main" > /etc/apt/sources.list.d/ros-latest.list'
+  - pip install colcon-common-extensions flake8 pytest PyYAML argparse
+  - curl -s https://raw.githubusercontent.com/ros/rosdistro/master/ros.asc | sudo apt-key add -
+  - sudo sh -c 'echo "deb http://packages.ros.org/ros2/ubuntu `lsb_release -cs` main" > /etc/apt/sources.list.d/ros2-latest.list'
   - sudo apt -qq update
-  - sudo apt -qq install -y ros-kinetic-catkin ros-kinetic-roslaunch ros-kinetic-rostest ros-kinetic-roslint
+  - sudo apt -qq install ros-ardent-ament-index-python ros-ardent-ament-cmake ros-ardent-ament-lint-auto ros-ardent-ament-cmake-pytest
```

Comment:
Using `rosdep` is a great idea. Unfortunately, we need to stick with `ardent`, because that's the only ROS2 version supported in Xenial (and Xenial is the "latest" Ubuntu distro supported by Travis).
Of course, we could run a `docker` container, but I would like to avoid this.

Reply:
@clalancette, I implemented the rosdep-based installation. However, rosdep doesn't know about ardent anymore ([see Travis](https://travis-ci.org/ros/xacro/jobs/590116041#L663-L667)).
I'm afraid we are stuck to ardent and manual installation.
We should enable the ROS build farm for CI testing as well when releasing.

Reply:
I'm pretty sure Travis supports Bionic: https://docs.travis-ci.com/user/reference/bionic/

Reply:
Thanks! I didn't look for a while...